.expeditor/verify_public.pipeline.yml

```diff
@@ -574,8 +574,9 @@ steps:
 
 - label: "Spellcheck"
   commands:
+    - ruby -rjson -e "JSON.parse(File.read('cspell.json'))" 2>/dev/null || (echo "Failed to parse config file 'cspell.json', skipping spellcheck" && exit 1)
     - npm install -g cspell
-    - cspell "**/*" "*.md"
+    - cspell "**/*"
```

Comment:
Everything in the `"*.md"` glob is also in the `"**/*` glob, so it caused those files to be spellchecked twice.